### ISSUE TITLE
function-space-after: "always" conflicts with multiple values

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#162]
Given:

```css
.selector-4 {
  color: color(rgb(0, 0, 0)lightness(50%)); 
  background-image: linear-gradient(#f3c, #4ec), linear-gradient(#f3c, #4ec); 
}
```

And `function-space-after: [2, "always"]`. We get:

```console
css\input.css
3:3     Expected single space after function's ")" (function-space-after) [stylelint]
4:3     Expected single space after function's ")" (function-space-after) [stylelint]
```

It correctly flags up the lack of space before `lightness` on line 3, but it also flags a warning because of the comma after the first `linear-gradient(#f3c, #4ec)` on line 4.

Should checking for and ignoring the comma of multiple values be the default behaviour, or should there be an additional option value e.g. `always-except-if-multi-values`?

**Comment by davidtheclark** [comment]
In my opinion, accepting commas as well as spaces should be the default. I would be surprised if somebody wanted to enforce `linear-gradient(#f3c, #4ec) , linear-gradient(#f3c, #4ec)`.
